### ISSUE TITLE
Research Staking/Unstaking/Slashing/Jackpot 

### ISSUE THREAD

**Issue by rtsisyk** [stegos/stegos#277]

**Comment by rtsisyk** [comment]
@dbmcclain I saw a draft of proposal. Could you please elaborate and write RFC in our [wiki](https://github.com/stegos/stegos/wiki) and/or update the yellow paper. I still don't have the full picture of this feature.

**Comment by dbmcclain** [comment]
I don't think any of us has a full picture of this yet. Vlad and Joel keep arguing about tokenomics in relation to these features.

**Comment by rtsisyk** [comment]
I don't think that tokenomics can affect Staking/Unstaking significantly. We need to implement a technology to stake and unstake tokens to/from escrow. All constants can be tuned later. 

Let's start by writing a spec/RFC for Staking/Unstaking UTXOs/Transactions.

**Comment by dbmcclain** [comment]
Sorry to disagree here. I can rough out a prelim design for us to argue about. But our decisions here affect incentives in the system, and that is the connection to tokenomics.
